README.md

```diff
@@ -1 +1,72 @@
-# recipe-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Blog app
+
+The Recipe app keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them.
+
+### Preview
+
+Coming soon
+
+## Live Version
+
+Coming soon
+
+## Built With
+- Ruby on Rails
+
+## Getting Started
+- Follow setup instructions below
+
+### Prerequisites
+
+- Ruby
+- Git and Github
+- Terminal
+- Browser
+
+### Setup
+
+- clone the following repo `https://github.com/orozCoding/recipe-app.git`
+- cd into the directory and run `bundle install` to install dependencies
+- Run `rails db:create` and then `rails db:migrate`
+- Run `rails s` to start the app
+- Visit `localhost:3000` in your browser
+
+### Tests
+
+- To run the tests, simply run `rspec` command
```

Comment:
- [ ] Great job showing how to run tests in you Readme, but you have not written tests. Kindly write `Unit and Integration tests.`